### ISSUE TITLE
SQL Formatting

### ISSUE THREAD

**Issue by lavanyadeepak** [OpenDBDiff/OpenDBDiff#134]
This could be taken as a feature request. When trying to move into another database is it possible to introduce a formatting of the scripts. Like when taking from local developer desktop to integration databases an option to neatly indent/format would be helpful for the developer.

Or if some one could give some ideas on how it can be implemented a fork of this could achieve and then we can submit a pull request.

**Comment by ghost** [comment]
yes, a kind of parsing the whole diff script would help better while running the whole diff script in the destination database.
